tailwind.config.ts

```diff
@@ -25,6 +25,8 @@ const config: Config = {
         white: "#FFFFFF",
         black: "#1C2434",
         red: "#FB5454",
+        "red-300": "#fca5a5",
```

Comment:
IMHO i think it isn't necessary add color, please check the colors offered by tailwind and let's try to stick with those